### PR TITLE
LSO - Mantis 24135 - check for extended tracking settings before showing first or last access

### PR DESCRIPTION
**Pull request by daniwe4**

### CODE REVIEW

File: setup/sql/dbupdate_05.php [hunk 1]
```diff
@@ -759,3 +759,28 @@
...
+
+<#5484>
+<?php
+include_once('./Services/Migration/DBUpdate_3560/classes/class.ilDBUpdateNewObjectType.php');
+
+$lp_type_id = ilDBUpdateNewObjectType::getObjectTypeId('lso');
+if ($lp_type_id) {
+	$ops_id = ilDBUpdateNewObjectType::getCustomRBACOperationId("lp_other_users");
+	ilDBUpdateNewObjectType::deleteRBACOperation($lp_type_id, $ops_id);
+	$ilDB->manipulate("DELETE FROM rbac_operations WHERE ops_id = ".$ilDB->quote($ops_id, "integer"));
```
Comment (klees) [inline setup/sql/dbupdate_05.php]: I do not think we want to do this. We did not add that permission in the first place, lets not remove it here.

**Review by klees — CHANGES_REQUESTED** [review]
Hi @daniwe4, thx a lot. This looks good so far. Please adjust that one db-update-step. Thx!

File: setup/sql/dbupdate_05.php (hunk 1 above)
Comment (daniwe4) [reply setup/sql/dbupdate_05.php]: Please have a look at update step 5480. There i put the permission lp_other_user for lso into rbac system. So why not delete it at this place?

**Comment by klees** [comment]
Picked to release_5-4 as 187c2574493f37d351d7679cca50b38af9921390.

**Pull request merged**